tests/integration/routes/accounts/get-accounts-test.js

```diff
@@ -54,8 +55,20 @@ getServer(function (error, server) {
       t.end()
     })
 
-    group.test('Not an admin', {todo: true}, function (t) {
-      t.end()
+    group.test('Not an admin', function (t) {
+      var requestOptions = _.defaultsDeep({
+        headers: {
+          authorization: 'Bearer cGF0LWRvZToxMjc1MDA6zEZsQ1BuO-W8SthDSrg8KXQ8OlQ'
```

Comment:
Does this line deserve a comment? I copied the authorization header from the `tests/integration/routes/get-account-test.js`, so I'm not really sure how to generate these tokens or how they relate to users in the pouchDB mock...